src/lib/log.c

```diff
@@ -37,6 +37,14 @@ static void log_internal(FILE *out, const char *file, int line, const char *labe
 	char time_str[50];
 	struct tm *timeinfo;
 	int printed;
+	const char *msg;
```

Comment:
we don't need a msg here. Just use the format variable.
It's ok to do a format++;

Reply:
ok I wasn't sure if modifying that pointer was ok, so I wanted to play it safe. I'll use the `format` pointer then